runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -130,6 +132,10 @@ class ClientSessionData
    int8_t  _inUse;  // Number of concurrent compilations from the same client 
                     // Accessed with compilation monitor in hand
    VMInfo *_vmInfo; // info specific to a client VM that does not change, nullptr means not set
+
+   // Per-client versions of the unloaded class and method addresses kept in J9PersistentInfo
+   TR_AddressSet _unloadedClassAddresses;
+   TR_AddressSet _unloadedMethodAddresses;
```

Comment:
We don't need _unloadedMethodAddresses at the server. The reason is that this is used only in `isInUnloadedMethod` which is only used in `invalidateEntryIfInconsistent` that looks at `entry->getPC()`. We already pass lists of unloaded classes from the client to server and clean-up the IP hashtable at the server (we invalidate all j9mthods for a class that gets unloaded)